### ISSUE TITLE
DAI Download cart - fixes needed, as we discussed

### ISSUE THREAD

**Issue by cneidle** [DCS-LCSR/ASL-DAI#387]
- Decoupling the activation of the XML buttons from the selection boxes
- Redrawing the screen not after each selection, but only after the "download selected" button has been clicked
- Replying to Kanakanti: (1) to email the 2 files that were not available on the site (PF18_V01_RIT-p2_10.xml and PF18_V01_RIT-p3_7.xml) ; and, when ready, (2) to reply about the site functionality having been fixed.

Thanks !!

**Comment by kwasiopoku** [comment]
This has been resolved and pushed to dev1 and dai

**Comment by cneidle** [comment]
The Collection buttons still seem to be inactive.

And it is impossible to check the “select” button.
(The check disappears within milliseconds.)

Happy to show you if you like…

Thanks,
 Carol

**Comment by cneidle** [comment]
What we agreed:

"The header for the column for downloading annotations will be changed to read:
Download SIgnStream collection and/or XML file."

The button will read "Annotations" rather than "Collection."

The Annotations button will be activated by checking something in either or both of the preceding columns.

Clicking on "download" (either through the annotations button or the button at the bottom) will uncheck the boxes that had been checked, add the downloaded items to the history, but will not redraw the page (e.g., causing the page
to jump to the top).

The region at the bottom will be modified to retain only what is in the first 4 columns (although the radio buttons should be removed from columns 3 and 4).

The listing should be sorted 1st by the name of the collection, and 2nd by download date.

Thank you !!

**Low priority potential improvement for the future:**

Add a column just before the video button with checkboxes, and make the video button
work like the annotation button, being activated only by being checked.

This would make it possible for someone to check the box for the XML file and the box for the
corresponding video.  When the annotations button has been selected, the checkbook would 
remain in the video box.  This would allow the user to download the annotations file first and
then the video file second.  Right now, the user is likely to lose track of what they are doing,
unless they download the video file before the annotations, because there will be no record
of the collections file they just downloaded so they can download the corresponding video.

**Comment by kwasiopoku** [comment]
The priority items for our discussion have all been completed and pushed to dev1 and dai

**Comment by cneidle** [comment]
Thank you.

But the Download Annotations button does not download.

It opens up a window like the one below.

The Download button at the bottom appears to work, but the
downloaded zip file cannot be opened.  I’m attaching that to this
message.  And the checkbox does not get unchecked after download.
Sorry!

Thanks again,
    Carol

PS. In case you care, the capital letter in second position of "S I gnstream"
should be changed to lower case:

 "Download SIgnStream collection and/or XML extract"

**Comment by cneidle** [comment]
thank you !!!!!!!!!
